src/gui/widgets/logbookwidget.cpp

```diff
@@ -54,6 +54,7 @@ LogbookWidget::LogbookWidget(QWidget *parent) :
 
 LogbookWidget::~LogbookWidget()
 {
+    delete displayModel;  // [G]: previously leaked from logbookwidget.cpp:89
```

Comment:
wouldn't it be easier to create the model with this as parent and let qt handle this?
```
89: displayModel = new QSqlTableModel(this);
```

Reply:
fixed